addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -22,7 +22,7 @@
                 Find a reseller <t t-foreach="countries" t-as="country"><span t-if="country['active'] and country['country_id'][0] != 0 and country['country_id']">in <t t-out="country['country_id'][1]"/></span></t>
             </h4>
             <div class="dropdown d-none d-lg-block">
-                <a role="button" href="#" data-bs-toggle="dropdown" t-attf-class="dropdown-toggle btn btn-light #{'d-none' if not partners else ''}" aria-expanded="true" aria-label="Open categories dropdown">
```

Comment:
This is not a stable change:
- First if you were to change a `t-attf-X`, keep it as a `t-attf-X`, only change to X in master
- Removing stuff from attribute values is never stable... but I guess there is no way around it if we really do want this change. Well, there is: you could just change this by **adding** a class, which forces the display back to not none if the class contains exactly "d-none fix_display_not_none" (e.g. `[class*="d-none fix_display_not_none"]`). And add a comment. That way you minimized the change the break custo as much as possible: if someone has an xpath targeting this to actually remove the `#{'d-none' if not partners else ''}`, it will still work. There are still custo that would break of course (like if someone actually wants to hide the button more. But maybe too much, I don't know.

The minimum IMO is to keep the `t-attf-` so xpath can still targeting it. Then it would be ideal to only **add** stuff inside and not **remove** so that xpath can still target by value without much risk. But the most ideal would be to do nothing... unless necessary but I don't see why this is necessary from the information here :grimacing: